pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.9</version>
+        <version>2.13</version>
```

Comment:
I've updated the parent pom version to latest as we discussed.
I did the update in this PR. I don't see the need for a separate PR for it.